web/js/codeworld_shared.js

```diff
@@ -39,6 +39,23 @@ function sendHttp(method, url, body, callback) {
     request.send(body);
 }
 
+function sendHttpWithProps(method, url, body, properties, callback) {
```

Comment:
Okay, but don't copy and paste.  Make sendHttp forward to this one with an empty map.